realm/realm-library/src/main/java/io/realm/rx/RealmObservableFactory.java

```diff
@@ -240,6 +272,7 @@ public void onChange() {
                     public void call() {
                         object.removeChangeListener(listener);
                         observableRealm.close();
+                        objectRefs.get().acquireReference(object);
```

Comment:
`objectRefs.get().acquireReference(object)` -> `objectRefs.get().releaseReference(object)`

I think you want to decrease the reference count here?